t/Unicode.t

```diff
@@ -18,6 +18,11 @@ BEGIN {
 	print "1..0 # Skip: Unicode support requires Windows 2000 or later\n";
 	exit 0;
     }
+    my $short = Win32::GetShortPathName(Win32::GetCwd().'\t\ExpandEnvironmentStrings.t');
+    if(length $short > 12) {
```

Comment:
This is not correct. Since you are using an absolute path, the length is almost guaranteed to be longer than 12.

Maybe you meant to check the relative path:

```
my $short = Win32::GetShortPathName('\t\ExpandEnvironmentStrings.t');
```

Reply:
I have a c:\ that supports short path names and a d:\ that does not. The resulting values of $short for three variations on this theme, run from the respective `.\perl5\cpan\Win32` folders, are shown below.

my $short = Win32::GetShortPathName(Win32::GetCwd().'\t\ExpandEnvironmentStrings.t');
```
c: C:\perl5\cpan\Win32\t\EXPAND~1.T
d: D:\perl5\cpan\Win32\t\ExpandEnvironmentStrings.t
```
my $short = Win32::GetShortPathName('\t\ExpandEnvironmentStrings.t');
```
c: undef
d: undef
```
my $short = Win32::GetShortPathName('t\ExpandEnvironmentStrings.t');
```
c: t\EXPAND~1.T
d: t\ExpandEnvironmentStrings.t
```
So the latter looks like the correct argument for GetShortPathName.